client/lib/plugins/utils.js

```diff
@@ -338,6 +338,7 @@ export function getPluginAuthorProfileKeyword( plugin ) {
  * @returns the correct business plan slug depending on current plan and pluginBillingPeriod
  */
 export function businessPlanToAdd( currentPlan, pluginBillingPeriod, eligibleForProPlan ) {
+	// @todo clk any rule for Starter plan?
```

Comment:
@claudiucelfilip familiar with these parts?

Reply:
Nope, I'm not.